### PR TITLE
Added publications tab and publication row vue components

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/js/vue/components/research-group/PublicationsTab.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        components: { PublicationRow },
+        props: {
+            datasets: {}
+        },
+        methods: {
+            getPublications: function () {
+                let publications = [];
+                let publicationId = [];
+                this.datasets.forEach(dataset => {
+                    dataset.publications.forEach(publication => {
```
Comment (praneethpr) [inline assets/js/vue/components/research-group/PublicationsTab.vue:20]: Not a fan of using nested FOR loops, couldn't find a better way. Please let me know if there is an efficient way to do this

Comment (mickel1138) [reply assets/js/vue/components/research-group/PublicationsTab.vue:20]: Maybe you could use `array_unique` ?

Comment (fightingtexasaggie) [reply assets/js/vue/components/research-group/PublicationsTab.vue:20]: The overhead of the inefficiency of the nested loops is negligible with the relative small numbers we're ever likely to deal with, and the inner loop isn't doing that much anyway. I'm ok with it as is, but Mike's suggestion would be fine too.

Comment (praneethpr) [reply assets/js/vue/components/research-group/PublicationsTab.vue:20]: Isn't array_unique a Php function?

**Pull request merged**